codepen-samples/braintree-client-sdk.md

```diff
@@ -10,7 +10,13 @@ Card Form Designs:
 
 [Simple](https://codepen.io/braintree/pen/ZWPpPG) | [Material](https://codepen.io/braintree/pen/mPgdPN) | [Minimal](https://codepen.io/braintree/pen/EKJVXp) | [Raw](https://codepen.io/braintree/pen/grQJQM) |  [3D Transform](https://codepen.io/braintree/pen/oLxqjd) | [Bootstrap v4](https://codepen.io/braintree/pen/zeamxM) | [Expiration Date Drop Down](https://codepen.io/braintree/pen/QGrXav)
 
-[3DS with Hosted Fields](https://codepen.io/braintree/pen/ezvymm)
+[3DS]
+- Drop in : https://codepen.io/vimalnath53/pen/zYNPQzZ
```

Comment:
Please create the links in the format: [Sample](https://www.example.com) separated by "|"
Example:
[Drop in](https://codepen.io/vimalnath53/pen/zYNPQzZ) | [Hosted Fields](https://codepen.io/vimalnath53/pen/vYxyKXL)